src/server/publicApi.js

```diff
@@ -143,7 +160,13 @@ const runPublicApi = (gameServer) => {
     return [app, appHandle]
 }
 
-
+function isSecretValid(ctx, metadata) {
+    if (metadata.players[ctx.params.id].credentials !== ctx.params.secret) {
+        ctx.status = 403;
```

Comment:
Oh, I see...  the http code is set here...

Actually, I wouldn't expect an `isSomething()` function to have such an side effect. Is there some better place to set the return code?
Maybe wrap the above `if`-blocks in something like an `assertValidSecrets()` function?

Reply:
I think it's probably clearest if I just set the return code in the `if(!isSecretValid())` statement